ow_lander/scripts/mechanical_power_arm.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
Change python to python2. We need to be specific for users who have python3 configured as the default on their computers.

Reply:
Done!